jobs/jobs.go

```diff
@@ -0,0 +1,32 @@
+package jobs
+
+import "github.com/mitchellh/mapstructure"
+
+type Type string
+
+const (
+	FetchResources             Type = "fetch_resources"
+	ExpiringGrantNotification  Type = "expiring_grant_notification"
+	RevokeExpiredGrants        Type = "revoke_expired_grants"
+	RevokeGrantsByUserCriteria Type = "revoke_grants_by_user_criteria"
+
+	// Deprecated: use RevokeExpiredGrants instead
+	RevokeExpiredAccess Type = "revoke_expired_access"
+	// Deprecated: use ExpiringGrantNotification instead
+	ExpiringAccessNotification Type = "expiring_access_notification"
```

Comment:
To make it clear it is a type constant, maybe we could prepend with `Type...` ? Would be easier to navigate with autocomplete too.

```suggestion
	TypeFetchResources             Type = "fetch_resources"
	TypeExpiringGrantNotification  Type = "expiring_grant_notification"
	TypeRevokeExpiredGrants        Type = "revoke_expired_grants"
	TypeRevokeGrantsByUserCriteria Type = "revoke_grants_by_user_criteria"

	// Deprecated: use RevokeExpiredGrants instead
	TypeRevokeExpiredAccess Type = "revoke_expired_access"
	// Deprecated: use ExpiringGrantNotification instead
	TypeExpiringAccessNotification Type = "expiring_access_notification"
```